main.py

```diff
@@ -315,26 +353,26 @@ def get_data_from_github(url, properties):
     # "retrieved" does only accept dates without time, so create a timestamp with no date
     # noinspection PyUnresolvedReferences
     isotimestamp = pywikibot.Timestamp.utcnow().toISOformat()
-    date = pywikibot.WbTime.fromTimestr(
-        isotimestamp, calendarmodel=Settings.calendarmodel
-    )
-    date.hour = 0
-    date.minute = 0
-    date.second = 0
-    date.precision = pywikibot.WbTime.PRECISION["day"]
-    properties["retrieved"] = date
+    properties["retrieved"] = string_to_wddate(isotimestamp)
 
     # General project information
     project_info = get_json_cached(github_repo_to_api(url))
 
     if project_info.get("homepage"):
         properties["website"] = project_info["homepage"]
 
-    releases = get_all_github_releases(url)
+    apiurl = github_repo_to_api_releases(url)
+    releases = get_all_pages(apiurl)
+    analyse = analyse_release
+    if Settings.read_tags and len(releases) == 0:
+        logger.info("Falling back to tags.")
+        apiurl = github_repo_to_api_tags(url)
+        releases = get_all_pages(apiurl)
+        analyse = analyse_tag
```

Comment:
I'd prefer not to create function-aliases here. Having both `[analyse_tag(tag, project_info) for release in tags]` and  `[analyse_release(release, project_info) for release in releases]` or something similar should work.

Reply:
Yeah, I already thought about that this not beeing very pythonic. I might have coded to much haskell recently. ;) Will try to figure something out.